src/main/java/frc/robot/commands/ShooterRangeTargetCommand.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.HubTracking;
+import frc.robot.subsystems.Shooter;
+import frc.robot.subsystems.HubTracking.HubData;
+
+public class ShooterRangeTargetCommand extends CommandBase {
+  Shooter shooter;
+  HubTracking hubTracking;
+  double range=0;
+  HubData data;
+
+  /** Creates a new ShooterRangeTargetCommand. */
+  public ShooterRangeTargetCommand(Shooter shooter_, HubTracking hubTracking_) {
+    shooter=shooter_;
+    hubTracking=hubTracking_;
+    data=new HubData();
+    addRequirements(shooter);
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    SmartDashboard.putBoolean("VISIBLE", hubTracking.isHubVisible());
```

Comment:
I think we need to specify that these are for the hub - so HUB IS VISIBLE ... is that useful in the command though?  Seems like the subsystem should set this.